docs/source/developer/contributing.rst

```diff
@@ -260,6 +261,11 @@ and then running::
 from the root of the Numba repository. Now ``flake8`` will be run each time
 you commit changes. You can skip this check with ``git commit --no-verify``.
 
+.. note::
+   To use these pre-commit hooks on Windows, it may be necessary to copy
+   ``sqlite3.dll`` from ``C:\Users\<USER>\anaconda3\Library\bin`` to
+   ``C:\Users\<USER>\anaconda3\DLLs`` or modify your PATH (`source <https://stackoverflow.com/a/55642416/13697228>`_).
+
```

Comment:
I can't verify this due to lack of a developmental machine, is there anyone who can confirm this work?

Reply:
I'm assuming anyone means core devs, but for reference, this was necessary for me to do on Windows to get pre-commit hooks working after following the contrib directions as closely as possible.

Reply:
Well, yes, core devs (on a loose definition of the term), would be ideal. But I think we would also be satisfied with anyone other than the author confirming this. Thank you!

Reply:
I can't replicate the issue on Windows - this seems like a little bit of an odd situation to have got into, and I'd suggest we shouldn't include the note on the basis that the linked source doesn't provide enough background about what's going on here for it to be clear that this is generally good advice.

Reply:
Ok, maybe something weird on my end, then. Sounds good!

Reply:
Are you planning to remove this note?

Reply:
Sorry, yes, forgot about this one.

Reply:
pre-commit hooks in a conda environment on Windows have still been a huge pain. Unfortunately, my workaround has just been to uninstall the hooks in many cases.